src/inputmethodkit.cs

```diff
@@ -165,27 +176,78 @@ partial interface IMKInputController : IMKStateSetting, IMKMouseHandling {
 		NSObject Delegate { get; set; }
 
 		[Export ("server")]
-		IMKServer Server { get; }
+		IMKServer Server ();
 
 		[Export ("client")]
-		NSObject Client { get; }
+		IIMKTextInput Client ();
```

Comment:
Same: why change it to a method?

Reply:
On Swift you can use Client() to get the result in exactly the IMKTextInput type.